packages/web-components/src/components/ic-status-tag/ic-status-tag.tsx

```diff
@@ -27,6 +27,10 @@ export class StatusTag {
    * If `true`, the small styling will be applied to the status tag.
    */
   @Prop() small?: boolean = false;
+  /**
+   * If 'true', the component will act as an 'aria-live' region.
```

Comment:
Is it worth mentioning it adds the `role=status` to the component as I was initially confused why it talks about aria-live?